tests/outputs/stmtTests.mg.out

```diff
@@ -1 +1,15 @@
 tests/inputs/stmtTests.mg:18:9: Error: function not in scope in tests.inputs.stmtTests.StmtTests: _=_ with arguments of type (T1, T2)
+
+tests/inputs/stmtTests.mg:33:9: Type error in tests.inputs.stmtTests.VarExprTests: got conflicting type annotation for var x: x has type T but type annotation is U
+
+tests/inputs/stmtTests.mg:56:29: Type error in tests.inputs.stmtTests.CallExprTests: could not deduce return type of call to overload_return_type. Possible candidates have return types [T, U]. Consider adding a type annotation
+
+tests/inputs/stmtTests.mg:68:9: Error in tests.inputs.stmtTests.LetExprTests: variable already defined in scope: x
```

Comment:
Is that error message clear enough? Perhaps we could improve it by writing something like `conflicting definitions for variable x (src1, src2)`? Not sure how easy it is to access the definition loc of the original variable when we throw the error. WDYT?

Reply:
Yes I agree it could be more clear. I can take a look at it in a separate PR.